omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js

```diff
@@ -78,12 +78,18 @@ function ConditionController($scope, RestfulService, ConditionModel, ConceptMode
 
     self.getSelectedDate = self.getSelectedDate || function () {
             var datePicker = angular.element(document.getElementsByName('conditionStartDate'))[0];
-            return datePicker.value;
+            var date = datePicker.value;
+            var formattedDate =date.replace(/-/g,'/');
+            var dateWithNoTime = new Date(formattedDate);
+            return dateWithNoTime.toUTCString();
         }
 
     self.getEndDate = self.getEndDate || function() {
             var endDatePicker = angular.element(document.getElementsByName('conditionEndDate'))[0];
-            return endDatePicker.value;
+            var date = endDatePicker.value;
+            var formattedDate =date.replace(/-/g,'/');
```

Comment:
Any reason for having the same exact block of code in more than one place?

Reply:
the first block is to handle the setDate while the second one is for endDate...Otherwise is your suggestion to write a new method and call it for these two instances?